contracts/FungibleToken.cdc

```diff
@@ -193,11 +192,30 @@ access(all) contract FungibleToken {
         }
     }
 
-    /// Global method to burn any FungibleToken Vault
+    /// Function to return the types that the contract implements
+    access(all) view fun getVaultTypes(): [Type] {
+        post {
+            result.length > 0: "Must indicate what fungible token types this contract defines"
+        }
+    }
+
+    /// createEmptyVault allows any user to create a new Vault that has a zero balance
+    ///
+    access(all) fun createEmptyVault(vaultType: Type): @{FungibleToken.Vault} {
```

Comment:
We decided on thursday that now, since contracts can define any number of tokens, the standard has to be able to support that by default, so contracts have to have a `createEmptyVault` function that specifies which of the defined tokens to create

Reply:
Yes